solr/contrib/prometheus-exporter/bin/solr-exporter

```diff
@@ -135,15 +134,42 @@ if $cygwin; then
   [ -n "$REPO" ] && REPO=`cygpath --path --windows "$REPO"`
 fi
 
+# Convert Environment Variables to Command Line Options
+EXPORTER_ARGS=()
+
+if [[ -n "$CONFIG_FILE" ]]; then
+  EXPORTER_ARGS+=(-f "$CONFIG_FILE")
+fi
+
+if [[ -n "$PORT" ]]; then
```

Comment:
I like the simplicity of `PORT`, but is there a risk that such generic naming could lead to conflict, i.e. that a user hast this env set for some other app, and then the exporter will use it?

Reply:
Yeah, that could certainly happen. `EXPORTER_PORT` would work perfectly fine. I went the direction decided in https://github.com/apache/lucene-solr/pull/887 on variable naming. I don't have a strong opinion either way.

Reply:
For a docker env it's not a problem, and for most servers you'll have a separate service user with clean shell. Developers could unintentionally have the PORT var set on their laptop, but I don't think we should make design decisions from that.

On a side note, I tested what would happen if you have both env `$PORT=1111` and arg `-p 2222`, and the last arg will win, thus you can override any env with explicit command arg, which I guess is a good thing.

Reply:
That is good to know. In that case we should probably stick with `$PORT`, as there's an easy fix in the case where someone is affected by it.